packages/commercetools/theme/components/Checkout/VsfShippingProvider.vue

```diff
@@ -53,6 +48,11 @@
         >
           {{ $t('Continue to billing') }}
         </SfButton>
+        <SfButton
+          class="sf-button--underlined form__back-button smartphone-only"
+          @click="$router.go(-1)"
```

Comment:
It won't work after you refresh the page. We have to come back to approach with a link

Reply:
When I think about it. As we do not have `Personal details` anymore - there is no `back`. So I think we could get rid of this btn